PotreeConverter/src/main.cpp

```diff
@@ -85,7 +85,7 @@ PotreeArguments parseArguments(int argc, char **argv){
 	args.addArgument("color-range", "");
 	args.addArgument("intensity-range", "");
 	args.addArgument("output-format", "Output format can be BINARY, LAS or LAZ. Default is BINARY");
-	args.addArgument("output-attributes,a", "can be any combination of RGB, INTENSITY, CLASSIFICATION, GPS_TIME, RTK_POSE, RTK_ORIENT, and DUAL_PLUS_CONFIDENCE. Default is RGB.");
+	args.addArgument("output-attributes,a", "can be any combination of RGB, INTENSITY, CLASSIFICATION, GPS_TIME, RTK_POSE, RTK_ORIENT, DUAL_PLUS_CONFIDENCE AND GEO_COORDINATES. Default is RGB.");
```

Comment:
`DUAL_PLUS_CONFIDENCE AND GEO_COORDINATES` => `DUAL_PLUS_CONFIDENCE, and GEO_COORDINATES`